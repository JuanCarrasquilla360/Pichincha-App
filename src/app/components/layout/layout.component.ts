import { Component } from '@angular/core';
import { CommonModule } from '@angular/common';
import { HeaderComponent } from '../header/header.component';
import { ContentComponent } from '../content/content.component';
import { RouterOutlet } from '@angular/router';

@Component({
  selector: 'app-layout',
  standalone: true,
  imports: [CommonModule,  HeaderComponent, ContentComponent, RouterOutlet],
  templateUrl: './layout.component.html',
  styleUrl: './layout.component.scss'
})
export class LayoutComponent {

}
