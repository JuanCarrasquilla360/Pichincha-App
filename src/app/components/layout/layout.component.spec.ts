import { ComponentFixture, TestBed } from '@angular/core/testing';

import { LayoutComponent } from './layout.component';

describe('LayoutComponent', () => {
  let component: LayoutComponent;
  let fixture: ComponentFixture<LayoutComponent>;
  let compiled: HTMLElement;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [LayoutComponent]
    })
    .compileComponents();
    
    fixture = TestBed.createComponent(LayoutComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  test('should create', () => {
    expect(component).toBeTruthy();
  });
  test('should match with snapshot', () => {
    expect(compiled).toMatchSnapshot();
  });
});
