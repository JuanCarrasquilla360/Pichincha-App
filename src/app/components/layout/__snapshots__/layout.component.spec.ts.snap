// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutComponent should match with snapshot 1`] = `
<div
  id="root1"
>
  <app-header>
    <header>
      <div
        class="logo"
      >
        <a
          href="/"
        >
          <img
            src="assets/img/logo-header.jpg"
          />
        </a>
      </div>
    </header>
  </app-header>
  <app-content>
    <div
      class="content"
    >
      <router-outlet />
      
    </div>
  </app-content>
  <router-outlet />
  
</div>
`;
